README.md

```diff
@@ -2,6 +2,9 @@
 
 Fable bindings for virtual-dom
 
+## Website
+[fable.io/fable-arch](http://fable.io/fable-arch/#/)
```

Comment:
I would have write [http://fable.io/fable-arch](http://fable.io/fable-arch/#/)

Just because I find it more common to see the "http:". What do you think ?